src/libraries/System.Private.CoreLib/src/System/ObsoleteAttribute.cs

```diff
@@ -48,5 +48,9 @@ public ObsoleteAttribute(string? message, bool error)
         public string? Message => _message;
 
         public bool IsError => _error;
+
+        public string? DiagnosticId { get; set; }
```

Comment:
Nit: You can change the above properties to follow the same style, e.g. `public string? Message { get; }`

Reply:
you mean remove the backing fields?

Reply:
yes